docs/checks.md

```diff
@@ -330,6 +330,31 @@ is therefore not a definitive indication that the project is at risk.
 **Remediation steps**
 - Integrate the project with OSS-Fuzz by following the instructions [here](https://google.github.io/oss-fuzz/).
 
+## LicenseCheck 
```

Comment:
ignore this part. Looks like a previous PR forgot to generate the docs from the yaml file...